src/loader.rs

```diff
@@ -377,7 +377,7 @@ fn load_map(
         )))
         .insert(TiledMapMarker);
 
-    #[cfg(feature = "rapier")]
+    #[cfg(any(feature = "rapier", feature = "avian"))]
```

Comment:
(minor) Maybe we could use the `physics`feature flag ?

Reply:
I changed it to feature = "physics" in the loader